core/src/main/scala/org/http4s/internal/Logger.scala

```diff
@@ -6,7 +6,7 @@
 
 package org.http4s.internal
 
-import cats.effect.Sync
+import cats.effect.kernel.Sync
```

Comment:
not sure if we want kernel imports or not, they are re-exported in `core`, but there has been some recent activity. @djspiewak can clarify I think

Reply:
More ideal to keep it as it was and import directly from core. Kernel isn't needed here since core is a dependency.

Reply:
Why does http4s `core` need a dependency on CE `core`?

Reply:
Fs2 already has one because of SyncIO